nuget/framework/nunit.nuspec

```diff
@@ -52,6 +60,12 @@ Supported platforms:
     <file src="bin/net-4.0/nunit.framework.xml" target="lib\net40" />
     <file src="bin/net-4.5/nunit.framework.dll" target="lib\net45" />
     <file src="bin/net-4.5/nunit.framework.xml" target="lib\net45" />
+    <file src="bin/netstandard13/nunit.framework.dll" target="lib\xamarin.ios" />
+    <file src="bin/netstandard13/nunit.framework.xml" target="lib\xamarin.ios" />
+    <file src="bin/netstandard13/nunit.framework.dll" target="lib\monoandroid" />
+    <file src="bin/netstandard13/nunit.framework.xml" target="lib\monoandroid" />
+    <file src="bin/netstandard13/nunit.framework.dll" target="lib\uap" />
+    <file src="bin/netstandard13/nunit.framework.xml" target="lib\uap" />
```

Comment:
A separate UWP dir here isn't strictly necessary, as it would pull in the netstandard1.3 build anyway - but given the problems, I wanted to be explicit about the platforms we wanted to support.